src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,15 @@
 import React from 'react';
-import './MoviesList.scss';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  movies: Movie[],
+};
+
+export const MoviesList: React.FC<Props> = ({ movies }) => (
+  <div className="movies">
+    {movies.map(movie => (
+      <MovieCard {...movie} />
```

Comment:
we not allowed to use spread operators. Use destructuring

Reply:
agree
you should specify in what fields you pass some values